docs/experiment/experiment_introduction.md

```diff
@@ -0,0 +1,23 @@
+# Experiment Introduction
+
+After a successful login you will be redirected to your personal landing page.
+From there, [navigate](../project/project_introduction.md#project-navigation) into your project of interest to see the project summary.
```

Comment:
NIT: Is the link here necessary? I think it might make the user jump to something they don't want to know at the moment.

Reply:
I'd keep it for now since it's better to be thorough. In theory the introduction should enable a user to do the task even if no project has been defined yet, therefore the linkage was included